opennms-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -20,6 +20,7 @@
   <session-config>
     <cookie-config>
       <http-only>true</http-only>
+      <comment>__SAME_SITE_STRICT__</comment>
```

Comment:
lol, magic strings to enable this, huh? (sigh)
(that sigh was for your need to do this, not directed at you)